src/shared/components/annotation/OncoKB.tsx

```diff
@@ -100,7 +101,12 @@ export default class OncoKB extends React.Component<IOncoKbProps, {}>
 
             if (this.showFeedback)
             {
-                oncoKbContent = this.feedbackModal(this.props.indicator);
+                oncoKbContent = (
```

Comment:
This is a minor fix related to the feedback modal. (OncoKB icon was disappearing upon click on feedback button)